esp8266/coffee-button/coffee-button.ino

```diff
@@ -5,36 +5,29 @@
 
 // edit the SSID & password to match your local network
 const char* ssid = "<insert_ssid_here>"; 
-const char* password = "<insert_pwd_here>"; 
+const char* password = "<insert_pwd_here>";
 const char* host = "hooks.zapier.com"; // edit the host adress, ip address etc. 
-String url = "<insert_hook_url>"; // edit the hook url
+String url = "<insert_hook_url>"; // edit the hook url 
```

Comment:
whitespace removed line 8, added line 10

Reply:
Yeah Arduino IDE is a bit of a pain with whitespaces, I'll correct